### PR TITLE
reaper for #95

### PR DESCRIPTION
**Pull request by JahlomP**
added ttl index to user collection

### CODE REVIEW

File: include/config.js [hunk 1]
```diff
@@ -134,6 +134,11 @@ var config = {
                 spec: {admin: DESC},
                 options: {}
             },
+            {
+                collection: 'user',
+                spec: { created: { type: Date, expires: 2592000000, default: Date.now }},
+                options: {}
+            },
```
Comment (brianhyder) [inline include/config.js]: See my comments on the issue #95

Comment (JahlomP) [inline include/config.js]: ok

**Comment by brianhyder** [comment]
Went ahead and put in the fix.  Closing the pull request out.

**Comment by JahlomP** [comment]
Thanks and sorry for not fixing it earlier

**Pull request closed without merging**